### DISCUSSION TITLE
MediaElement IOS update default seek forward time

### DISCUSSION THREAD

**Discussion by igorsky78** [CommunityToolkit/Maui#1650] [General]
Currently, the default Forward/Backward seek time is 10 seconds for IOS. I want to update Forward seek time to 30 seconds. 

Is there a way in the MediaElment to update the default Forward/Backward seek time for IOS?

![image](https://github.com/CommunityToolkit/Maui/assets/8582700/7e528c64-cbaf-49ca-ae7b-1675e51d5149)

**Comment by CliffAgius** [comment]
@ne0rrmatrix is this possible or in the works?  Be nice to close this or answer if it can't be done. Thanks

**Reply by ne0rrmatrix** [reply]
The short answer is no. The long answer is we could create a custom control that would allow us to do this. But we would have to give up a lot. We just do not have access to things like full screen as there is no API for it. I believe there are other things to we do not have access to and apple will not even respond to dev questions about it.
